gradle-junit-reports/src/main/java/com/palantir/gradle/junit/XmlReportFailuresSupplier.java

```diff
@@ -55,7 +55,11 @@ private XmlReportFailuresSupplier(
     @Override
     public List<Failure> getFailures() throws IOException {
         File sourceReport = reporting.getReports().findByName("xml").getDestination();
-        return XmlUtils.parseXml(reportHandler, new FileInputStream(sourceReport)).failures();
+        try {
+            return XmlUtils.parseXml(reportHandler, new FileInputStream(sourceReport)).failures();
+        } catch (Exception e) {
+            throw new RuntimeException(String.format("Failed to parse failures XML: %s", sourceReport), e);
+        }
```

Comment:
I snuck this one in as well as checkstyle crashing (on files from resources) caused an unfinished xml to be written, and it wasn't obvious where that XML was